mmv1/products/netapp/storagePools.yaml

```diff
@@ -0,0 +1,233 @@
+
+# Copyright 2023 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+# API resource name
+name: 'storagePools'
```

Comment:
I named it as a plural because that's how it's being referred to in all the API's.
e.g. self_link: projects/{{project}}/locations/{{location}}/storagePools/{{storage_pool_id}}
create_url: projects/{{project}}/locations/{{location}}/storagePools?storagePoolId={{storage_pool_id}}
update_url: projects/{{project}}/locations/{{location}}/storagePools/{{storage_pool_id}}.
Let me know if that's not the correct reference, I'll change it to singular.

Reply:
I believe singular is also what we use for other resources in Terraform even if it's plural in the API: https://github.com/shuyama1/magic-modules/blob/main/mmv1/products/alloydb/Cluster.yaml.
Also see examples in the contribution guide: https://googlecloudplatform.github.io/magic-modules/develop/resource/#add-a-resource

Reply:
Updated the code with singular product name in all the relevant files.